### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54880]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Audit UX Improvements

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
When a user has a benefits application in progress (test user 220), the heading level for the application-in-progress card is an H4. The heading before that is an H2 so it skips a heading level, which can be disorienting for screen reader users.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
Change the heading level for the application-in-progress card to an H3.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by joshkimux** [comment]
<img width="1792" alt="Screenshot 2023-05-01 at 8 35 18 AM" src="https://user-images.githubusercontent.com/14154792/235452742-7554aba5-7c04-4682-a734-cca9dc514bb3.png">

I came across this issue too! Attached a screenshot in case it's helpful.

**Comment by ajakabcin** [comment]
Closing - #60625 should be out in prod now and this issue is resolved
